acra/security-controls/intrusion-detection/_index.md

```diff
@@ -2,3 +2,86 @@
+
+For intrusion detection, Acra uses poison records, also known as honey tokens.
+
+## Poison records
+
+Poison records are records specifically designed to sit quietly in the database and not be queried by legitimate users under normal circumstances. They look like any other encrypted records, and it's impossible to distinguish them from `normal data`. Technically speaking, poison records are data (**binary**, **strings**, **int** or whatever suits your database design), placed in particular tables/columns/cells.
+
+However, poison records will only be included in the outputs of requests from malicious applications that read more data than they should, i.e. using `SELECT *` requests. The goal of using poison records is simple — to detect adversaries trying to download full tables/database from the application server or trying to run full scans in their injected queries. AcraServer will inform user (system administrator) of untypical behaviour and can block suspicious requests.
+
+For more technical details you can check out other blog posts related poison records:
+
+- [Explain Like I’m Five: Poison Records (Honeypots for Database Tables)](https://hackernoon.com/poison-records-acra-eli5-d78250ef94f)
+- [Poison Records In Acra – Database Honeypots For Intrusion Detection](https://www.cossacklabs.com/blog/acra-poison-records.html)
+
+
+## Command line flags
+
+AcraServer has a set of flags for more precise work configuration with Poison records.
+
+
+* `--poison_detect_enable={true|false}`
+
+  Turn on poison record detection, if server shutdown is disabled, AcraServer logs the poison record detection and returns decrypted data.
+  Default is `true`.  
+
+
+* `--poison_shutdown_enable={true|false}`
+
+  On detecting poison record: log about poison record detection, stop and shutdown.
+  Default is `false`.
+
+* `--poison_run_script_file=<scriptpath>`
+
+  On detecting poison record: log about poison record detection, execute script, return decrypted data.
+
+
+## Usage example
+
+
+AcraServer's key storage contains a special key, which is used for recognition of poison records. This key is generated either after a query passes through AcraServer, or upon running poison record generation utility [`AcraPoisonRecordMaker`]({{< ref "acra/configuring-maintaining/general-configuration/acra-poisonrecordmaker" >}}) (but you will have to move the keys into AcraServer's key directory manually).
+
+To install [`AcraPoisonRecordMaker`]({{< ref "acra/configuring-maintaining/general-configuration/acra-poisonrecordmaker" >}}), run:
```

Comment:
imho, will be nice to refer to installation section from binaries instead of sources and place link.